apps/concierge_site/lib/controllers/unsubscribe_controller.ex

```diff
@@ -0,0 +1,19 @@
+defmodule ConciergeSite.UnsubscribeController do
+  use ConciergeSite.Web, :controller
+  alias AlertProcessor.Model.Trip
+  @thirty_days_in_seconds 2_592_000
+
+  def update(conn, %{"id" => encrypted_user_id}) do
+    # We're not using a separate salt here because we're not
+    # encrypting passwords and don't need the extra security
+    {:ok, user_id} =
+      Plug.Crypto.decrypt(conn.secret_key_base, conn.secret_key_base, encrypted_user_id,
```

Comment:
question: In non-test environments, is this `secret_key_base` value being added to the `conn` automatically? It looks like it's also an application environment variable, but if it's part of the conn, too, that's fine, I'm just not familiar with it.

Reply:
It apparently does read the secret_key_base out of the config (if available) and puts it in the conn automatically, so I could get it from either. I can't find anything about that in the documentation, but it works fine in my local dev environment.